packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -238,7 +238,7 @@ async function getAuthHandler(module) {
 
     // We use `version` to make sure we install the same version of the auth
     // setup package as the rest of the RW packages
-    await execa.command(`yarn add -D ${module}@${version}`, {
+    await execa.command(`yarn add -W -D ${module}@${version}`, {
```

Comment:
I don't see this `-W` flag listed in the [yarn docs](https://v3.yarnpkg.com/cli/add) for yarn v3. I do however see it in the [yarn docs](https://classic.yarnpkg.com/en/docs/cli/add) for v1. We would encourage using yarn v3 if you are not.

Do you mind confirming what version of yarn your project is using? I think `yarn -v` should provide that information.

Reply:
Ah! I am using yarn v1. I will remove this change as it probably only applies to folks not on the latest version.